docs/usage-examples/find.txt

```diff
@@ -0,0 +1,82 @@
+.. _laravel-find-usage:
+
+=======================
+Find Multiple Documents
+=======================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: find many, retrieve, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can retrieve multiple documents from a collection by chaining a retrieval
+method such as ``where()`` with the ``get()`` method to retrieve the results.
+You can chain these methods to an Eloquent model or a query builder.
```

Comment:
To give the reader more perspective, we can tell them that `Model::where()` creates a query builder. And that they can use all the other query builder methods, such as `orWhere`, `whereAll` ...
With a link to the [query builder page](https://github.com/mongodb/laravel-mongodb/blob/4.1/docs/query-builder.txt).

Reply:
Let me know what you think about [the new description I committed](https://github.com/mongodb/laravel-mongodb/pull/2833/commits/a4cd130549577a119eb1a1c9edf9ec6186ed4797)

If you're ok with this version, I'll create an item in our [cleanup ticket](https://jira.mongodb.org/browse/DOCSP-38344) to update the Usage Example descriptions.